### DISCUSSION TITLE
Adding Gitpod to the project

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1343] [Ideas]
[Gitpod](https://www.gitpod.io/) is an open source platform for automated and ready-to-code development environments that blends into your existing workflow. Gitpod is basically a Docker container allowing you to use a VS Code environment in the browser.  @Miodec I think integrating the Monkeytype repository with Gitpod would be useful because it allows users to develop and test new code without need to setup anything on their local machine(I use it to contribute on a chromebook, as I usually don't have access to my windows laptop and am not allowed to clone other's code locally on it for security reasons) this makes it easy for new contributors to get started. Here is a [repository](https://github.com/freeCodeCamp/freeCodeCamp) that currently uses Gitpod. If you're interested, here's how you can get [started](https://www.gitpod.io/docs/getting-started).

**Comment by tajhans** [comment]
Can't you open Gitpod by using the `gitpod.io/#` prefix? Like `gitpod.io/#https://github.com/Miodec/monkeytype`.
And I don't know but I feel like this seems a little bit unnecessary because how many people are going to use it? But I guess there isn't that much that goes into setting it up.

**Reply by unrealapex** [reply]
Yes it is possible to open a repo with Gitpod like `gitpod.io/#https://github.com/Miodec/monkeytype`. As far as usefulness of Gitpod, I think the ability to contribute in the browser with minimal setup is a big advantage that should make getting your feet wet very easy.

**Reply by tajhans** [reply]
Yeah that does make sense I guess.

**Reply by unrealapex** [reply]
We could have one configured to install firebase and run `npm i` as soon as the user opens it, then all the user needs to do is configure Firebase.

**Reply by unrealapex** [reply]
@teddinotteddy I don't think this is relevant, but go to the code tab of this repo and try pressing <kbd>.</kbd> 👍 :)
